modules/service/query-service.js

```diff
@@ -38,7 +38,13 @@ class QueryService {
             Id_operation: operationId,
         });
 
-        const { nquads, isAsset } = await this.dataService.resolve(id);
+        let isAsset = false;
+        const { assertionId } = await this.blockchainService.getAssetProofs(id);
```

Comment:
Note for future: Can we implement here some more lightweight method in smart contract, just to have isAsset

Reply:
Yes, it's already in the works